### ISSUE TITLE
width/height mixup in high-res images

### ISSUE THREAD

**Issue by jpriebe** [qooxdoo/qooxdoo#9599]
On line 519 of Image.js:

```
  var imageWidth = ResourceManager.getImageHeight(source);
  var imageHeight = ResourceManager.getImageWidth(source);
```

This causes images with high dpi versions (@2x versions) to have their height/width flipped, leading to some very strange results (unless, of course, the image is perfectly square, in which case you would never know).

**Comment by johnspackman** [comment]
@jpriebe Good catch!  Would you like to submit a PR 😉 and we'll integrate it? 👍

**Comment by jpriebe** [comment]
Sure.  I was just being lazy.   Any plans for a 5.0.3 release in the near future?

**Comment by johnspackman** [comment]
great, thank you.  don't know about 5.0.3 just yet - we have collected a set of essential fixes together before and back ported them into a release but we typically recommend using master (aka 6.0.0 release candidate).  i've asked the team what they think

**Comment by cboulanger** [comment]
Fixed with #9602
